Notation Command

SonofGod1998 opened the pull request:
Added a notation command: !notation

SonofGod1998 committed:
Notation Command

Added a notation command: !notation

JustAPenguin9 commented on src/commands/mod.rs:
```diff
@@ -7,3 +7,4 @@ pub mod online;
 pub mod owner;
 pub mod query;
 pub mod wiki;
+pub mod notation;
```
remove

JustAPenguin9 commented on src/commands/links.rs:
```diff
@@ -42,3 +42,12 @@ pub async fn repository(ctx: Context<'_>) -> Result<(), Error> {
+/// Sends a Notation image for AoCF
+#[poise::command(prefix_command, slash_command)]
+pub async fn notation(ctx: Context<'_>) -> Result<(), Error> {
+	ctx.reply("https://wiki.koumakan.jp/images/aocf/f/fd/AoCF-Notations.jpg")
+	.await?;
```
no newline needed, run `cargo fmt` before committing

JustAPenguin9 commented on src/commands/links.rs:
```diff
@@ -42,3 +42,12 @@ pub async fn repository(ctx: Context<'_>) -> Result<(), Error> {
 	Ok(())
 }
+
+/// Sends a Notation image for AoCF
```
"Notation" does not need to be capitalized and the wording could be better

JustAPenguin9 commented on src/main.rs:
```diff
@@ -84,6 +84,7 @@ async fn main() {
 				links::starthere(),
 				links::links(),
 				links::repository(),
+				links::notation(),
```
if the notation command is moved to `wiki.rs` this will need to become `wiki::notation()`

JustAPenguin9 commented on src/commands/links.rs:
```diff
@@ -42,3 +42,12 @@ pub async fn repository(ctx: Context<'_>) -> Result<(), Error> {
 }
+
+/// Sends a Notation image for AoCF
+#[poise::command(prefix_command, slash_command)]
+pub async fn notation(ctx: Context<'_>) -> Result<(), Error> {
+	ctx.reply("https://wiki.koumakan.jp/images/aocf/f/fd/AoCF-Notations.jpg")
+	.await?;
+
+	Ok(())
+}
```
the notation command should be in `wiki.rs` not `links.rs` since it deals with game knowledge

SonofGod1998 committed:
Add notation command

Moved notation command from links.rs to wiki.rs, changed main.rs from links to wiki and removed pub mod in mod.rs

JustAPenguin9 committed:
Fix errors
